wrappers/golang/internal/generator/main.go

```diff
@@ -46,33 +46,33 @@ var bn254 = CurveData{
 	PackageName:    "bn254",
 	Curve:          "bn254",
 	GnarkImport:    "bn254",
-	ScalarLimbsNum: 8,
-	BaseLimbsNum:   8,
-	G2BaseLimbsNum: 16,
+	ScalarLimbsNum: 4,
+	BaseLimbsNum:   4,
+	G2BaseLimbsNum: 8,
```

Comment:
Just wondering: can this `CurveData` struct accommodate curves without G2 (like grumpkin)?

Reply:
Currently no. The rest of the generation script for the curves also doesn't handle this case.
I think the entire generation script here needs to be a bit more modular in general and especially to support curves that don't implement all the functionality

Reply:
Right, cos grumpkin and some other curves also don't implement NTT (Nova doesn't need it) and we'll get to small fields hopefully soon where you don't have curves at all. But all that is outside of the scope of this PR ofc, just wanted to mention